target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -301,6 +302,9 @@ ramips_setup_macs()
 		[ -n "$lan_mac" ] || lan_mac=$(cat /sys/class/net/eth0/address)
 		wan_mac=$(macaddr_add "$lan_mac" 1)
 		;;
+	hootoo,tm05)
+		wan_mac=$(mtd_get_mac_binary factory 0x2e)
+		;;
```

Comment:
Since this is a one-port device, specifying a wan MAC address won't have any effect. So, remove this case here. However, if there is a real address at this flash location, please report that in the commit message (or even better, provide an overview of MAC addresses in your commit message.)

Reply:
OK, removed. Will rebuild and test here, confirm MAC's as expected.

As for the commit message - yes, you had mentioned this, so I had added it (and making sure it's in there after rebase).

Reply:
With this removed, I get (ifconfig) ... OK?
```
br-lan    Link encap:Ethernet  HWaddr 00:1C:C2:43:F6:B4
eth0      Link encap:Ethernet  HWaddr 00:1C:C2:43:F6:B4
wlan0     Link encap:Ethernet  HWaddr 00:1C:C2:43:F6:B4
```

Reply:
Yes. But it's still here.

Reply:
Removed here - will force a push again, see if that clears it up.